plugins/core/jetbrains-community/src/icons/AwsIcons.kt

```diff
@@ -143,6 +143,30 @@ object AwsIcons {
         @JvmField val CHECKMARK_GRAY = load("/icons/resources/codetransform/checkmark.svg")
 
         @JvmField val TIMELINE_STEP = load("/icons/resources/codetransform/transform-timeline-step-done.svg") // 16 * 16
+
+        @JvmField val PLAN_VARIABLES_LIGHT = load("/icons/resources/codetransform/transform-variables-light.svg")
```

Comment:
Just as a safety check, have we tried creating builds and testing that these icons are loaded correctly(aka not a sandbox run)? We've had instances in the past where the path did not load and the images weren't displayed

Reply:
Yes, this worked fine